### PR TITLE
Modify new session page CSS to match polaris

### PR DESCRIPTION
**Pull request by meulmees**
The styles for the app login page was fairly out-dated, and also didn't really work well on mobile.

### Web
screen shot 2017-09-16 at 4 18 50 pm

### Mobile
screen shot 2017-09-16 at 4 18 37 pm

### Web
screen shot 2017-09-16 at 4 18 02 pm

### Mobile
screen shot 2017-09-16 at 4 18 15 pm

I've added the App's logo within the title since I think this will help with branding.
I've also added the partners logo as a link at the top of the page which is consistent with the Identity login page and I think helps add branding/assurance that this is a Shopify backed product.

### CODE REVIEW

**Comment by meulmees** [comment]
@megnrosen - can I please get a design/content 👁 on this

**Comment by kevinhughes27** [comment]
This repo is for the base framework of **all** ShopifyApps both built by Shopify and built by partners. I am all for updating this install page to marketing assets but it needs to remain generic. If we want additional custom branding on this page for our own app we need to overwrite this view in that code base.

File: app/views/shopify_app/sessions/new.html.erb [hunk 1]
```diff
@@ -3,92 +3,143 @@
...
+        <path fill="#95BF46" d="M107.5 27.1c-.1-.7-.7-1.1-1.2-1.1s-11.2-.8-11.2-.8-7.4-7.4-8.3-8.2c-.8-.8-2.4-.6-3-.4-.1 0-1.6.5-4.2 1.3C77.1 10.7 72.7 4.2 65 4.2h-.7C62.1 1.3 59.4 0 57 0 39 0 30.4 22.5 27.7 33.9c-7 2.2-12 3.7-12.6 3.9-3.9 1.2-4 1.3-4.5 5C10.2 45.6 0 124.5 0 124.5l79.5 14.9 43.1-9.3c.2-.1-15-102.3-15.1-103zm-32.3-7.9c-2 .6-4.3 1.3-6.7 2.1v-1.5c0-4.4-.6-8-1.6-10.9 4 .5 6.6 5 8.3 10.3zM62 9.8c1.1 2.8 1.8 6.7 1.8 12.1v.8c-4.4 1.4-9.1 2.8-13.9 4.3C52.6 16.7 57.6 11.7 62 9.8zm-5.4-5.1c.8 0 1.6.3 2.3.8C53.1 8.2 47 15 44.3 28.7c-3.8 1.2-7.5 2.3-11 3.4C36.4 21.7 43.7 4.7 56.6 4.7z"></path>
+        <path fill="#5F8E3E" d="M106.3 25.9c-.5 0-11.2-.8-11.2-.8s-7.4-7.4-8.3-8.2c-.3-.3-.7-.5-1.1-.5l-6 123 43.1-9.3s-15.1-102.3-15.2-103c-.2-.7-.8-1.1-1.3-1.2z"></path>
+        <path fill="#FFF" d="M65 49.7l-5.3 15.8S55 63 49.3 63c-8.4 0-8.8 5.3-8.8 6.6 0 7.2 18.8 10 18.8 26.9 0 13.3-8.4 21.9-19.8 21.9-13.7 0-20.6-8.5-20.6-8.5l3.7-12.1s7.2 6.2 13.2 6.2c4 0 5.6-3.1 5.6-5.4 0-9.4-15.4-9.8-15.4-25.3 0-13 9.3-25.6 28.2-25.6 7.2 0 10.8 2 10.8 2z"></path>
+      </svg>
```
Comment (kevinhughes27) [inline app/views/shopify_app/sessions/new.html.erb]: We should probably remove this SVG

**Comment by meulmees** [comment]
@kevinhughes27 - good point.

I've removed the partners logo and app icon.
### Web
screen shot 2017-09-18 at 11 17 11 am

### Mobile
screen shot 2017-09-18 at 11 17 24 am

If/when we want to customize this, like you said, we can do this within our code base.

**Pull request merged**